pyi.Dockerfile

```diff
@@ -1,26 +1,44 @@
-# TODO: copied from the main Dockerfile... too bad we don't have includes.
-FROM ubuntu:20.04 as bcc-builder
+# Centos 7 image is used to grab an old version of `glibc` during `pyinstaller` bundling.
+# This will allow the executable to run on older versions of the kernel, eventually leading to the executable running on a wider range of machines.
+FROM centos:7 AS build-stage
 
-RUN apt-get update
+# bcc part
+# TODO: copied from the main Dockerfile... but modified a lot. we'd want to share it some day.
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y git build-essential iperf llvm-9-dev libclang-9-dev \
-  cmake python3 flex bison libelf-dev libz-dev
+RUN yum install -y \
+    git \
+    cmake \
+    python3 \
+    flex \
+    bison \
+    zlib-devel.x86_64 \
+    ncurses-devel \
+    elfutils-libelf-devel
 
 WORKDIR /bcc
 
 RUN git clone --depth 1 https://github.com/Granulate/bcc.git && cd bcc && git reset --hard 119d71bf9681182759eb76d40660c0ec19f3fc42
```

Comment:
It is tagged, I think. But I prefer hardcoding the commit because tags/branches may change, but this commit refers to exactly one revision, forever. I prefer it for hardcoding versions.